tools/justfile

```diff
@@ -117,9 +111,7 @@ deploy: build-all
   just compose-invoke queue
   just compose-invoke object
   just compose-invoke seq
-
   just compose-up
-
   wait
```

Comment:
What does it is meant to do?

Reply:
To make sure mongo is deployed before trying to populate the Db, but it is not necessary, I just removed it